README.md

```diff
@@ -116,9 +116,9 @@ Refer to the [docs folder](docs) for design overview and other info on Mobius
 |Run Mobius Shell |<ul><li>[Local](notes/mobius-shell.md#run-shell)</li><li>[YARN](notes/mobius-shell.md#run-shell)</li></ul> | Not supported yet |
 
 ### Useful Links
-* [Configuration parameters in Mobius](/notes/configuration-mobius.md)
-* [Troubleshoot errors in Mobius](/notes/troubleshooting-mobius.md)
-* [Debug Mobius apps](/notes/running-mobius-app.md#debug-mode)
+* [Configuration parameters in Mobius](./notes/configuration-mobius.md)
```

Comment:
Just curious how did you find this link broken? Quick check shows that this link (and some other links with similar changes) is not broken on GitHub.